src/main.rs

```diff
@@ -183,3 +170,33 @@ async fn main() {
         tracing::error!(%err, "server error");
     }
 }
+
+fn main() {
+    let opts: Opts = argh::from_env();
+
+    // this has to be defined before tokio multi-threads
+    let timer = OffsetTime::local_rfc_3339().expect("local time offset");
+    if opts.debug {
+        // use colorful log for debug
+        let output = tracing_subscriber::fmt::layer().compact();
+        tracing_subscriber::registry()
+            .with(
+                output
+                    .with_filter(filter::filter_fn(|metadata| {
+                        !metadata.target().starts_with("hyper")
+                    }))
+                    .with_filter(filter::LevelFilter::DEBUG),
+            )
+            .init();
+    } else {
+        // use JSON format for production
+        tracing_subscriber::fmt()
+            .with_max_level(tracing::Level::INFO)
+            .json()
+            .with_timer(timer)
```

Comment:
no need to explicitly add timer in debug mode?

Reply:
Should be a bug, let me fix it.

Reply:
Fixed. PTAL